### DISCUSSION TITLE
If the administrator email include upper-case letters it will be impossible to login

### DISCUSSION THREAD

**Discussion by TheFrenchGhosty** [requarks/wiki#3505] [Error / Bug Report]
**Describe the bug**
If the administrator email include upper-case letters, it will always be invalid when login in.

**To Reproduce**
1. Deploy an instance
2. Set the administrator email to one that include upper-case letter
3. Complete the setup in any way you want
4. Try to login with the email that include upper-case letters = it fails
5. Try to login with the email with the upper-case letters replaced with lower-case letter = it fails

**Expected behavior**
It works

**Host Info (please complete the following information):**
 - OS: Docker
 - Wiki.js version:  2.5.159 
 - Database engine: postgres:11-alpine

**Additional context**
Related (maybe): #2558

I just wasted ~2 hours because of that thinking my docker-compose was wrong.

**Comment by valumar** [comment]
Same issue for me. Version 2.5.170
